org.cubocore.CoreImage.appdata.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should consider maintaining this file in your git repo as it's not specific to Flathub.

Reply:
I will maintain it in git. Thanks for the pointing.

Reply:
Will it be a matter to store the `appdata.xml` to github/flathub repo for now until the app got another release?